src/core/layouts/Navigation/NavigationProject/NavigationProject.tsx

```diff
@@ -12,10 +11,13 @@ import { NavigationProjectLogo } from "./NavigationProjectLogo"
 
 const NavigationProjectWithQuery = () => {
   const projectSlug = useParam("projectSlug", "string")
-  const [project] = useQuery(getProjectWithId, {
-    slug: projectSlug!,
+  const [{ sections }] = useQuery(getSections, {
+    where: {
+      project: {
+        slug: projectSlug!,
+      },
+    },
```

Comment:
Das würde ich später umbauen wollen in Richtung https://github.com/FixMyBerlin/trassenscout/blob/ffd82a3bd752776fc6bd0a3f0d37f493219e25cd/src/stakeholdernotes/queries/getStakeholdernotes.ts#L16, so dass `getSections` zwingend mit projectSlug aufgerufen werden muss.